content/zh/docs/tasks/kubelet-credential-provider/kubelet-credential-provider.md

```diff
@@ -0,0 +1,201 @@
+---
+title: 配置 Kubelet 镜像凭证提供程序
+description: 配置 Kubelet 镜像凭证提供程序插件
+content_type: task
+---
```

Comment:
please keep the English version of the frontmatter in a comment block.

Reply:
The English comment must come after the localization in this case. I.e. move line 1-10 after line 15.